client/hosting/logs/index.tsx

```diff
@@ -10,8 +10,8 @@ import {
 } from 'calypso/hosting/sites/components/site-preview-pane/constants';
 import { siteDashboard } from 'calypso/hosting/sites/controller';
 import { siteSelection, sites, navigation } from 'calypso/my-sites/controller';
-import { redirectHomeIfIneligible } from 'calypso/my-sites/site-monitoring/controller';
-import { httpRequestLogs, phpErrorLogs } from './controller';
+import { redirectHomeIfIneligible } from '../monitoring/controller';
+import { phpErrorLogs, webServerLogs } from './controller';
 
 export default function () {
```

Comment:
Just out of curiosity - who run this function, to initiate routes?

Reply:
Should be here:

https://github.com/Automattic/wp-calypso/pull/94524/files#diff-bbfd0752d7fc9a4e930a58d3f153a24662dd06baa00a529caf8b7715e9477003R737